api/v0/proto/retrievability/options.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package celestia.v0.retrievability.options;
+
+enum IncludeProof {
+  INCLUDE_PROOF_UNSPECIFIED = 0;
+  // Include the proof with the data.
+  INCLUDE_PROOF_YES = 1;
+  // Don't include the proof with the data.
+  INCLUDE_PROOF_NO = 2;
+  // Include only the proof, not the data.
+  INCLUDE_PROOF_ONLY = 3;
+}
+
+// Common options used across services.
+message Options {
+  // Whether to include the proof with the data, or not, or just the proof.
+  IncludeProof include_proof = 1;
+}
```

Comment:
might no longer be needed as message. How about to add ` optional IncludeProof` everywhere instead?

Reply:
That's probably clearer, especially since it doesn't look like there's any need for any other options that we can think of.

Reply:
Fixed in 441125d8e93b7fc88bca58b41f0278aa239837bf and 10e89cc4626f9d0bfd1137b87c18792c8d8d548d